### ISSUE TITLE
Type of tool - add CI tools

### ISSUE THREAD

**Issue by AndrewArch** [w3c/wai-selecting-eval-tools#35]
[ED-MED]
Location: Main Features
Type of tool: I notice that we don’t have “Continuous Integration” tools listed here (or in the database filter. Noting that there seemed to be some CI tools in the database, e.g. the IBM tools.)
Consider mentioning here (and as a filter option in the database), e.g.
Current: Evaluation tools can be plugins (extensions) for authoring tools and web browsers, command line tools, desktop or mobile applications, and online services.
Proposed: Evaluation tools can be plugins (extensions) for authoring tools and web browsers, command line tools, continuous integration tools, desktop or mobile applications, and online services.

**Comment by kakinney** [comment]
is it just text somewhere or are we adding a new filter?

**Comment by shawna-slh** [comment]
<from quick skim; apologies if I missed something>

"Type of tool" is
* Filter in https://www.w3.org/WAI/test-evaluate/tools/list/
* listed in Selecting https://www.w3.org/WAI/test-evaluate/tools/selecting/#features

I assume those lists should match?

So this issue is not just about adding CI to the selecting doc -- also need:
* to be added in the submission for
* JSON updated for relevant tools
* probably added to the filters of the tool list

Correct?

**Comment by brianelton** [comment]
@shawna-slh yes, that makes sense. I assume @iadawn needs to make these updates?

**Comment by shawna-slh** [comment]
ok, I assume not high priority
